.github/workflows/ci.yaml

```diff
@@ -32,39 +33,39 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: check
-        args: --all --bins --examples --features hyperium_http
+        args: --workspace --benches --bins --examples --tests --features hyperium_http
 
     - name: check unstable
       uses: actions-rs/cargo@v1
       with:
-        command:  check
-        args: --all --benches --bins --examples --tests --features "hyperium_http,unstable"
+        command: check
+        args: --workspace --benches --bins --examples --tests --features "hyperium_http,unstable"
 
     - name: tests
       uses: actions-rs/cargo@v1
       with:
         command: test
-        args: --all --features "hyperium_http,unstable"
+        args: --workspace --features "hyperium_http,unstable"
 
-  check_fmt_and_docs:
+  check_fmt_clippy_docs:
     name: Checking fmt, clippy, and docs
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-    - uses: actions-rs/toolchain@v1
-      with:
-        toolchain: nightly
-        components: rustfmt, clippy
-        override: true
+
+   - name: beta setup
+      run: |
+        rustup toolchain install beta
+        rustup default beta
```

Comment:
Can you say more why using `beta` makes sense here?

Reply:
Oh I guess it's just to get us off nightly?

Reply:
Yeah, nightly is currently missing `rustfmt`...